runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/InMemoryArtifactStagerService.java

```diff
@@ -70,7 +77,12 @@ public void commitManifest(
   }
 
   public Manifest getManifest() {
-    return manifest;
+    return manifest.get();
+  }
+
+  public void clear() {
```

Comment:
This is a new method that isn't called. Did you mean to include some more files in this PR?

Reply:
Oh no I have failed to cleanup the temporary change that allowed me to run the test many times.

RM'd